Tests/UnitTests/Purchasing/OfferingsManagerTests.swift

```diff
@@ -495,6 +495,7 @@ private extension OfferingsManagerTests {
                     Offering(
                         identifier: offering.identifier,
                         serverDescription: offering.description,
+                        metadata: [:],
```

Comment:
The other tests only check `OfferingsResponse`. Let's add some metadata here so you can check in another test in this file that they get copied correctly to `Offering` (basically that`offering.metadata.mapValues(\.asAny)`).